mdetsims/defaults.py

```diff
@@ -70,3 +74,135 @@
     # check for an edge hit
     'bmask_flags': 2**30,
 }
+
+TEST_METACAL_MOF_CONFIG = {
```

Comment:
Looking through this config would be useful.

Reply:
this looks consistent with what I expect